suites/quincy/rados/tier-2_rados_ec-pool_recovery.yaml

```diff
@@ -27,6 +27,8 @@ tests:
                 registry-url: registry.redhat.io
                 mon-ip: node1
                 allow-fqdn-hostname: true
+                rhcs-version: 6.1
+                release: rc
```

Comment:
What happens if the test suite is executed for 6.1 RC build itself? Shall we consider using an older z-stream version here, something which is not dynamic?
6.1 z-stream -> upgrade to 6.1 RC -> upgrade to 6.1 latest nightly